mc/Mongo-Core.package/Mongo.class/instance/openConnectionToHost.port.timeout..st

```diff
@@ -0,0 +1,12 @@
+operations
+openConnectionToHost: aString port: anInteger timeout: timeoutInSeconds
+	| result |
+	result := self socketStreamClass
+		openConnectionToHost: aString
+		port: anInteger
+		timeout: timeoutInSeconds.
+	self tls
+		ifTrue: [ result sslSession serverName: self host. result connect ]
+	ifFalse: [ result socket setOption: 'TCP_NODELAY' value: 1 ].
```

Comment:
Did you have any trouble using "TCP_NODELAY=1" with TLS connection? maybe it's an option for both cases (TLS enabled and disabled).

IIRC, @zecke added that lien because it was much more efficient for the scenario of a db. But I may be wrong, of course.

Reply:
Found this: https://web.archive.org/web/20160319034601/https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_MRG/1.2/html/Realtime_Tuning_Guide/sect-Realtime_Tuning_Guide-Application_Tuning_and_Deployment-TCP_NODELAY_and_Small_Buffer_Writes.html